org-roam.el

```diff
@@ -870,6 +870,26 @@ whose title is 'Index'."
         (concat (expand-file-name org-roam-directory) path)
       index)))
 
+;;;; dealing with file-wide properties
+(defun org-roam--set-global-prop (name value)
+  "Set a file property called NAME to VALUE.
+
+If the property is already set, it's value is replaced."
+  (save-excursion
+    (widen)
+    (goto-char (point-min))
+    (if (re-search-forward (concat "^#\\+" name ": \\(.*\\)") (point-max) t)
+        (replace-match (concat "#+" name ": " value) 'fixedcase)
+      ;; find the first line that doesn't begin with ':' or '#'
+      (let ((found))
+        (while (not (or found (eobp)))
+          (beginning-of-line)
+          (if (or (looking-at "^#")
+                  (looking-at "^:"))
+              (line-move 1 t)
+            (setq found t)))
```

Comment:
```suggestion
(while (and (not (eobp))
            (looking-at "^[#:]"))
  (forward-line)
  (beginning-of-line))
```

Reply:
Just for explain this diff:

1. `line-move` runs a lot of things, and depends on some user config options (like whether to create new lines). `forward-line` is the simplest way to go one line down

2. We need to explicitly check the boundary condition so we know when to create a new line, since `forward-line` does not error

Reply:
Makes sense! Thank you.